src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -48,6 +48,8 @@ ContextAttribute::ContextAttribute()
   value                 = "";
   compoundValueP        = NULL;
   typeFromXmlAttribute  = "";
+  providingApplication  = "";
+  found                 = true;
```

Comment:
Confirming that all constructors contain the setting of 'found' and 'providingApplication'.